### PR TITLE
Improve the package request page to support giving only a ticket id

### PR DESCRIPTION
**Pull request by pypingou**
Fixes https://github.com/fedora-infra/pkgdb2/issues/229

### CODE REVIEW

File: pkgdb2/templates/package_request.html [hunk 1]
```diff
@@ -44,12 +43,20 @@
             document.body.style.cursor = 'wait';
             $('#pkgname').val('');
             $('#summary').val('');
+            $('#pkgname').prop('disabled', true);
+            $('#summary').prop('disabled', true);
             var _bzurl = $('#review_url').val();
             var _bzid = null;
-            if (_bzurl.indexOf("?id=") > -1) {
-                _bzid = _bzurl.split("?id=")[1];
+            var _bzidint = false;
```
Comment (ralphbean) [inline pkgdb2/templates/package_request.html]: It looks like you set ``_bzidint`` here, but then never use it.

Comment (pypingou) [inline pkgdb2/templates/package_request.html]: true, I had an idea but turned out to be un-necessary, will drop it

File: pkgdb2/templates/package_request.html [hunk 2]
```diff
@@ -44,12 +43,20 @@
             document.body.style.cursor = 'wait';
             $('#pkgname').val('');
             $('#summary').val('');
+            $('#pkgname').prop('disabled', true);
```
Comment (ralphbean) [inline pkgdb2/templates/package_request.html]: If you're disabling pkgname and summary, should you disable/reenable the review url field too?

Comment (pypingou) [inline pkgdb2/templates/package_request.html]: disable when checking, re-enable it at the end? Good idea

File: pkgdb2/templates/package_request.html [hunk 3]
```diff
@@ -44,12 +43,20 @@
...
+            $('#summary').prop('disabled', true);
+            $('#review_url').prop('disabled', true);
             var _bzurl = $('#review_url').val();
             var _bzid = null;
-            if (_bzurl.indexOf("?id=") > -1) {
-                _bzid = _bzurl.split("?id=")[1];
+            if ($.isNumeric(_bzurl)) {
+                _bzid = _bzurl;
+                _bzidint = true;
```
Comment (ralphbean) [inline pkgdb2/templates/package_request.html]: Still setting ``_bzidint`` here.

**Comment by ralphbean** [comment]
OK, :+1:

**Comment by pypingou** [comment]
Thanks for the review!

**Pull request merged**